model/task/task.go

```diff
@@ -1264,6 +1252,12 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 	return nil
 }
 
+// GetTaskIdBetweenIds retrieves the task id between two tasks that are of the same.
+func GetTaskIdBetweenTasks(t1, t2 Task) (*Task, error) {
```

Comment:
This could imply that it would get any task ID in between the revisions for t1 and t2. Can we make this function more explicit that it gets the task midway between the two?